authentication/views.py

```diff
@@ -33,7 +34,26 @@ def login_page(request):
 	return render(request, html)
 
 def sign_up(request):
-	return HttpResponse("Sign up beech")
+
+	registered = False
+
+	if request.method =="POST":
+		user_form = SignupForm(data=request.POST)
+		if user_form.is_valid():
+			user_form.save()
+			raw_password = user_form.cleaned_data.get('password1')
+			username = user_form.cleaned_data.get('username')
+			user = authenticate(username=username, passowrd=raw_password)
+			login(request,user)
+			# Not currently working: return redirect(sign_up)
```

Comment:
I thought we fixed this on skype

Reply:
Whoops, forgot to push after making those changes